### DISCUSSION TITLE
Has anyone successfully used --information from the command line?

### DISCUSSION THREAD

**Discussion by playsbyear** [gpx-animator/gpx-animator#576] [Q&A] [answered]
I get this error message:
[main] ERROR app.gpx_animator.Main - Missing implementation of command line option 'INFORMATION'. Please report to support.

Just want to make sure this is a real issue with the software and not my installation before I open a ticket. I am able to successfully use many other command line options (--input, --color, --information-postion, etc.)

**Comment by playsbyear** [comment, marked as answer]
I downloaded/executed the current source code and see that this is getting fixed in the next release. Thanks!

**Comment by McPringle** [comment]
Yes, you are right. I hope it won't take long to get the next release ready.
